.github/workflows/merge-tests.yml

```diff
@@ -136,15 +136,38 @@ jobs:
         uses: peter-evans/commit-comment@v3
         with:
           body-path: BENCHMARKS.md
-      # TODO: Use jq for JSON parsing if needed
-      # Check for benchmark regression based on Criterion's configured noise threshold
-      - name: Performance regression check
-        id: check-regression
-        run: echo "regress_count=$(grep -c 'Regressed' ${{ github.sha }}.json)" | tee -a $GITHUB_OUTPUT
-      # Fail job if regression found
-      - uses: actions/github-script@v7
-        if: ${{ steps.check-regression.outputs.regress_count > 0 }}
-        with:
-          script: |
-            core.setFailed('Fibonacci bench regression detected')
+      # Check for a slowdown >= 10%. If so, open an issue but don't block merge
+      - name: Check for perf regression
+        id: regression-check
+        run: |
+          regressions=$(awk -F'[*x]' '/slower/{print $11}' BENCHMARKS.md)
+
+          echo $regressions
 
+          for r in $regressions
+          do
+            if (( $(echo "$r >= 0.10" | bc -l) ))
```

Comment:
Would this trigger on https://github.com/lurk-lab/lurk-rs/commit/d372ffd19d5c4f6da22518d3aed3c5f573b2dae8#comments, which contains "1.01x slower", given that 1.01 > 0.10?